test/server/TypeDBCoreRunner.java

```diff
@@ -43,18 +42,13 @@ protected int port() {
         return port;
     }
 
-    @Override
-    protected void verifyPortUnused() {
-        verifyPortUnused(port());
-    }
-
     @Override
     protected List<String> command() {
         List<String> command = new ArrayList<>();
         command.addAll(getTypeDBBinary());
         command.add("server");
         command.add("--server.address");
-        command.add("0.0.0.0:" + port);
+        command.add(address());
```

Comment:
use the "127.0.0.1:<port>" address to connect, instead of a random hardcoded "0.0.0.0"